pkg/utils/load.go

```diff
@@ -0,0 +1,75 @@
+/*
+ Copyright 2022 The KubeVela Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ 	http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package utils
+
+import (
+	"context"
+	j "encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/oam-dev/kubevela/pkg/utils/common"
+)
+
+// ReadRemoteOrLocalPath will read a path remote or locally
+func ReadRemoteOrLocalPath(pathOrURL string, saveLocal bool) ([]byte, error) {
```

Comment:
pkg/utils/file.go LoadDataFromPath looks similar to this function?

Reply:
they're not exactly the same, not related with this PR